client-mvp-04/src/components/createAccount/createAccountContainer.test.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import CreateAccountContainer from './createAccountContainer';
+import { MemoryRouter } from 'react-router-dom';
+import { cleanup, fireEvent, render, screen, wait } from '@testing-library/react';
+import userReducer from '../../store/reducers/userReducer';
+import service from '../../services/user.service';
+
+// Mock Redux Store
+const mockStore = configureStore([]);
+let store = mockStore({
+    user: userReducer,
+});
+store.dispatch = jest.fn();
```

Comment:
Your component doesn't connect to redux store. Please, remove the Mock Redux Store (10-15 lines).